examples/AlwaysOnJaegerExample.php

```diff
@@ -56,6 +56,12 @@
             'id' => md5((string) microtime(true)),
         ]));
 
+        try {
```

Comment:
Did you mean to leave these in the example? Same question for the other example.